src/StackAdmin/Azs.Compute.Admin/Tests/src/Quota.Tests.ps1

```diff
@@ -216,10 +222,26 @@ InModuleScope Azs.Compute.Admin {
                 Get-AzsComputeQuota -Location | Where-Object { $_.Name -eq $name} | Should be $null
             }
         }
-
-        It "TestUpdateQuota" -Skip:$('TestUpdateQuota' -in $global:SkippedTests) {
-            $global:TestName = 'TestUpdateQuota'
-            Set-AzsComputeQuota -Location $global:Location -Name "UpdateQuota" -AvailabilitySetCount 100 -CoresLimit 100 -VmScaleSetCount 100 -VirtualMachineCount 100
+		
+		# Session recording for this needs a manual update.
+		# Set command would try to do a get before to ensure the quota exists.
+        It "TestQuotaCreateUpdateDelete" -Skip:$('TestQuotaCreateUpdateDelete' -in $global:SkippedTests) {
+            $global:TestName = 'TestQuotaCreateUpdateDelete'
+
+			#
+			#	if running against the actual environment enable the following to create first.
+			#	New-AzsComputeQuota -Location $global:Location -Name "testQuotaCreateUpdateDelete" -AvailabilitySetCount 1 -CoresCount 1 -VmScaleSetCount 1 -VirtualMachineCount 1 -StandardManagedDiskAndSnapshotSize 1 -PremiumManagedDiskAndSnapshotSize 1
```

Comment:
You should have recorded the create.

Reply:
for some reason recording with powershell is not working where as with sdk did work well.